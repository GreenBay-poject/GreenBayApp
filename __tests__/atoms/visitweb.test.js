import React from 'react';
import renderer from 'react-test-renderer';
import { act } from 'react-test-renderer';
import VisitWeb from '../../atoms/visitweb'

describe('<VisitWeb />', () => {
    it('Sample VisitWeb', () => {
        const tree = renderer.create(<VisitWeb url={"http://www.google.com"} />).toJSON();
        expect(tree).toMatchSnapshot();
        expect(tree.type).toBe('View')
        expect(tree.children.length).toBe(1);
        expect(tree.children[0].type).toBe('View');
        expect(tree.children[0].children[0].type).toBe('Text');
    });
});