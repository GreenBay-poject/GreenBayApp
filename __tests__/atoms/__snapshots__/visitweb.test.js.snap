// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VisitWeb /> Sample VisitWeb 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#0e0463",
        "borderTopLeftRadius": 100,
        "borderTopRightRadius": 100,
        "height": 60,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#FFFFFF",
          "fontSize": 19,
          "fontWeight": "bold",
          "marginTop": 15,
        }
      }
    >
      VISIT OUR WEB
    </Text>
  </View>
</View>
`;
